lib/govuk_publishing_components/presenters/machine_readable/article_schema.rb

```diff
@@ -69,20 +70,21 @@ def publishing_organisation
       end
 
       def is_part_of
-        return {} unless linked_step_by_steps.any?
+        return {} unless step_by_step_schemas.any? || collection_pages("document_collections").any?
+
```

Comment:
using `collection_pages("document_collections")` here and then `document_collections` below is a little confusing and duplicates code. If someone changed that the bits you want are somewhere other than "document_collections" then you'd have to change that in two places (here and in `document_collections`). I'd suggest DRYing it up by changing it to `document_collections.any?`

Reply:
👍